modules/apps/portal-search-solr7/portal-search-solr7-impl/src/main/java/com/liferay/portal/search/solr7/internal/search/engine/adapter/search/SearchSolrQueryAssemblerImpl.java

```diff
@@ -253,7 +253,12 @@ protected void setSorts(
 
 			SolrQuery.ORDER order = SolrQuery.ORDER.asc;
 
-			if (sort.isReverse() || sortFieldName.equals("score")) {
+			String originalSortName = sort.getFieldName();
+
+			if (sort.isReverse() ||
+				(sortFieldName.equals("score") &&
+				 !sortFieldName.equals(originalSortName))) {
```

Comment:
Remove this check - see comment at https://github.com/liferay-search/liferay-portal/pull/10/files#diff-0fd9f35f159ee8c153ccbb56758aeb2cR76